### ISSUE TITLE
Saving model

### ISSUE THREAD

**Issue by Eric-Simon-IA** [xuhongzuo/couta#2]
Hello,
First, thank you for your work, I've been using USAD so far but your solution seems to give similar or even better results on my data, while being easier to tune/train 

I'm not really familar with models built that way, so I'm kinda stuck on one point, if you could guide me I'd be grateful:
I trained my model on colab, but need to make it available on another computer (inference on CPU only) for experimentation purpose.

Here is the struggle : I Can't find the proper way to save the model to be used elsewhere. 

What I tried :  
- dill/pickle/torch.save the trained model object, rearrange functions,... but there's always at least one of the dependencies that cannot be interpreted. I know it's not the best pratice to do so anyway
- Just saved the 'net' state_dict, recreate the COUTA instance and load dict but too much things are done in fit(), so I can't reload weights if the model is not fitted before

Any help will be appreciated, thank you in advance

**Comment by Eric-Simon-IA** [comment]
Well I managed to make something work now, by saving state_dict, dim and c, but not sure if it's the best way to do that

**Comment by xuhongzuo** [comment]
Thanks for raising this issue. I will add model saving and loading into the notebook file.   
Yes, saving `state_dict` of the network, the center `c`, and the dimensionality `dim` seems to be a good solution.

**Comment by xuhongzuo** [comment]
Hi @Eric-Simon-IA, I just added `model save and load` to `couta_algo.py`. You can now assign `save_model_path` and `load_model_path` when initializing COUTA to realize saving the trained network or loading of the fitted network. I also update a section in the `Readme.md`.

Thanks again for raising this issue and sorry for the late update.  
I will close this issue, and if you have any further problems related to this problem, please feel free to reopen it. 

Cheers! Hope you have a wonderful week!

**Comment by Eric-Simon-IA** [comment]
Thank you, will look at it when I manage to get some free time.
Hope you have a nice week as well !
